frontend/src/scenes/retention/RetentionTable.tsx

```diff
@@ -98,7 +98,7 @@ export function RetentionTable({ dashboardItemId = null }: { dashboardItemId?: s
                         minWidth: results[selectedRow]?.values[0]?.count === 0 ? '10%' : '90%',
                         fontSize: 16,
                     }}
-                    title={results[selectedRow] ? moment(results[selectedRow].date).format('MMMM d, YYYY') : ''}
+                    title={results[selectedRow] ? moment.utc(results[selectedRow].date).format('MMMM D, YYYY') : ''}
```

Comment:
Sort of related to this PR:

Noticed yesterday that retention table has almost no typing (e.g. results is an array of any). Would it make sense to improve this as part of this refactoring? Since the types might make it easier to catch something being wrong.

Reply:
yeah i was looking at another error and noticed this too. giving it an update right now!

Reply:
Put those changes in #3091